### DISCUSSION TITLE
Playlists

### DISCUSSION THREAD

**Discussion by rishanbenahalkar** [SudhanPlayz/Discord-MusicBot#984] [Ideas]
**Is your feature request related to a problem? Please describe.**
A playlists command, you can make playlists, add songs to your playlists, basically how spotify has playlists, you could delete them, clone them, share them, etc. 

**Describe the ideal solution**
This helps add an array of songs to the queue instead of manually copying the link of each song in your Spotify playlist, MySQL can be used to store such data, although I'm very unsure if jsoning can do it, either way since Darren mentioned in a previous post that eventually MySQL will be implemented, this can be kept on hold until a database to store such data is implemented.

**Describe alternatives you've considered**
None at the moment

**Additional context**
null

**Comment by JotaroKujo0525** [comment]
Will move this into discussion
